src/components/Searchbar/Searchbar.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from "react";
+import ResultField from "./ResultField";
+import CloseButton from "./CloseButton";
+import atractions from "./dummyData"; // podmień na dumm data z projektu kiedy powstanie
+import "./searchbar.css";
+
+const Searchbar = () => {
+  const [searchInput, setSearchInput] = useState("");
+  const [atractionList, setatractionList] = useState([]);
+
+  const handleChange = (e) => {
+    setSearchInput(e.target.value);
+    setatractionList(atractions);
+  };
+
+  const clearInput = (e) => {
+    setSearchInput("");
+  };
+
+  return (
+    <>
+      <div className="searchbar">
+        {/* input field  */}
+        <input
+          className="searchbar__input"
+          value={searchInput}
+          placeholder={"Wyszukaj po nazwie miejsca.."}
+          onChange={handleChange}
+        />
+        {/* result field */}
+        {searchInput.length > 0 ? (
```

Comment:
Instead of a ternary operator, we could use logical AND. If the first part is false, then the second isn't returned, therefore nothing isn't rendered there.

Reply:
tak, to dobre